base/io_buf.h

```diff
@@ -65,6 +65,15 @@ class IoBuf {
     size_ += sz;
   }
 
```

Comment:
By the way, there is no reason for IoBuf to be immovable, but this requires implementing the move constructor manually because we can't have a default-default constructor (as the constructor with default value capacity=256) conflicts with it

Reply:
Have not put too much thought.into it. Feel free to extend if it blocks you in Dragonfly